source/system/structs/VectorList.ooc

```diff
@@ -51,9 +51,11 @@ VectorList: class <T> extends List<T>{
 		this _count -= 1
 		result
 	}
-	removeAt: override func (index: Int) {
-		this _vector copy(index + 1, index)
-		this _count -= 1
+	removeAt: override func (index: Int) { this removeRange(index, index + 1) }
```

Comment:
The point is that one-liners are OK if there's more than one of them together. Switching between the two styles too often impacts readability.

This is ok
```ooc
a: func { lala }
b: func { lala }
c: func {
    lolo
    lolo
}
d: func {
    lolo
    lolo
}
```
This is not
```ooc
a: func { lala }
b: func {
    lolo
    lolo
}
c: func { lala }
d: func {
    lolo
    lolo
}
```
We do break this rule a lot though, especially if, for example, `a` and `b` have the same name and `a` only calls `b` with some default arguments.